data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,851 @@
+
+
+outfit "Ballistic Cannon Turret"
+	category "Turrets"
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	cost 1370000
+	thumbnail "outfit/ballistic turret"
+	"mass" 120
+	"outfit space" -120
+	"weapon capacity" -120
+	"turret mounts" -1
+	"required crew" 3
+	weapon
+		sprite "projectile/ballistic"
+		"hardpoint sprite" "hardpoint/ballistic"
+		"hardpoint offset" -1
+		sound "ballistic"
+		"fire effect" "ballistic fire"
+		"live effect" "ballistic smoke" 36
+		"hit effect" "ballistic hit"
+		"inaccuracy" 3
+		"turret turn" 0.6
+		"velocity" 8
+		"lifetime" 100
+		"reload" 110
+		"firing energy" 247
+		"firing heat" 537
+		"shield damage" 145
+		"hull damage" 776
+		"hit force" 360
+		"missile strength" 18
+	description `As the largest caliber cannon in the Gegno Vi's arsenal, Ballistic Cannons require a lot of work to be fitted on even the largest of ships. A huge, self-sustaining base with its own vent ports is necessary for one of these cannons to even be fitted on a turret mount, provided there are crew to operate it. Combining destructive force with adjustable aim, Ballisic Cannon Turrets are the deadliest weapons of Vi warships.`
+
+effect "ballistic fire"
+	sprite "effect/balfire"
+		"frame rate" 15
+		"no repeat"
+	"lifetime" 25
+
+effect "ballistic hit"
+	sprite "effect/explosion/medium"
+		"frame rate" 15
+		"no repeat"
+	sound "explosion huge"
+	"lifetime" 36
+	"random angle" 360
+	"random frame rate" 5
+	"random velocity" 1
+	"random spin" 5
+	"velocity scale" 0.4
+	
+effect "ballistic smoke"
+	sprite "effect/smoke"
+		"scale" 0.1
+		"no repeat"
+		"frame rate" 10
+	"lifetime" 30
+	"random spin" 6
+	"random angle" 360
+	"random velocity" 0.5
+	"velocity scale" 0.01
+
+outfit "Astuit Battery"
+	category "Turrets"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 764000
+	thumbnail "outfit/astuit"
+	"mass" 28
+	"outfit space" -28
+	"weapon capacity" -28
+	"turret mounts" -1
+	weapon
+		sprite "projectile/astuit"
+		"hardpoint sprite" "hardpoint/astuit"
+		"hardpoint offset" 8
+		sound "astuit"
+		"hit effect" "astuit hit"
+		"inaccuracy" 5
+		"turret turn" 2.2
+		"velocity" 8
+		"lifetime" 44
+		"reload" 6
+		"firing energy" 13.2
+		"firing heat" 34
+		"shield damage" 8.8
+		"hull damage" 9.4
+	description "The Gegno Scin are the first Gegno to advance plasma technology to the point where it can be used in combat, although the energy requirement of such technology is still significant. Astuit Batteries work very similar to human Energy Blasters, firing small condensed plasma projectiles at high rates of fire. With four blasters on a turreted mount, it produces a constant stream of damage which adds up over time, despite the minor damage that each individual projectile creates."
```

Comment:
Is there an individual "Astuit blaster" ? This description talks about Astuit batteries repeatedly, then after talking about energy blasters, is suddenly talking about putting four blasters on a turret, which makes it sound like it shifted focus to putting four energy blasters on the turret. A little confusing.

If there is an "astuit blaster", then this should be reworded a bit so that it doesn't sound like it is talking about energy blasters. If there isn't, then I'd suggested changing the term to "astuit cannon" to avoid confusion with energy blasters. Alternatively, just use their full name. This is what I have in the suggestion box:

```suggestion
	description "The Gegno Scin are the first Gegno to advance plasma technology to the point where it can be used in combat, although the energy requirement of such technology is still significant. Astuit Batteries work very similar to human Energy Blasters, firing small condensed plasma projectiles at high rates of fire. With four Astuit Blasters on a turreted mount, it produces a constant stream of damage which adds up over time, despite the minor damage that each individual projectile creates."
```

Reply:
There currently is not Astuit blaster, but there might be in the future? Not sure really. But yes this looks better.